modules/kn-service-describe.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-describe_{context}"]
+= Describing serverless applications
+
+You can describe a Knative service by using the `kn` CLI `kn service describe` command.
+
+.Example commands
+
+* Describe a service:
++
+[source,terminal]
+----
+$ kn service describe -v <service_name>
+----
++
+The `-v` verbose flag is optional but can be included to provide a more detailed description. The difference between a regular and verbose output is shown in the following examples:
```

Comment:
In the example you reference `--verbose` (long option), here `-v` (short option). Both is possible, but maybe you should stick to one version ?